### PR TITLE
Tree

### PR DESCRIPTION
**Pull request by daisyjanejohnson**
- [x ] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ x] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code

### CODE REVIEW

File: Data-Structures/trees/__TEST__/tree.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  it('can successfully return a collection from a postorder traversal', () => {
+    // const apples = new Node('apples', 'bananas', 'cherries');
+    const banana = new Node('banana');
+    const cherries = new Node('cherries');
+    const apple = new Node('apples', banana, cherries);
+    const tree = new BinarySearchTree(apple);
+
+    expect(tree.postOrder()).toEqual(['banana', 'cherries', 'apples']);
+  });
```
Comment (adrianhuebner) [inline Data-Structures/trees/__TEST__/tree.test.js:63]: What I would do to do a test that searches for a specific node is to create a tree, then use that contains method. Then you can expect it to be false or true with whatever value you want to use.
